tests/UnpackerTest.php

```diff
@@ -12,6 +12,16 @@
 use Symfony\Flex\PackageResolver;
 use Symfony\Flex\Unpack\Operation;
 use Symfony\Flex\Unpacker;
+use function class_exists;
+use function file_get_contents;
+use function file_put_contents;
+use function getenv;
+use function json_decode;
+use function mkdir;
+use function putenv;
+use function unlink;
+use function var_dump;
```

Comment:
We don't import global functions in Symfony. Please don't change the coding standards of the project.